oauth2_http/java/com/google/auth/oauth2/UserCredentials.java

```diff
@@ -129,7 +129,31 @@ public AccessToken refreshAccessToken() throws IOException {
     return access;
   }
 
+  /**
+   * Returns client ID of the credential from the console.
+   *
+   * @return client ID
+   */
+  public final String getClientId() {
+    return clientId;
+  }
+
+  /**
+   * Returns client secret of the credential from the console.
+   *
+   * @return client secret
+   */
+  public final String getClientSecret() {
+    return clientSecret;
+  }
+
+  /**
+   * Returns the refresh token resulting from a OAuth2 consent flow.
+   *
+   * @return refresh token
+   */
   public final String getRefreshToken() {
     return refreshToken;
   }
+
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace seems to neither encourage nor discourage whitespace after the last member of the class.
To keep things the same, I'll remove it before merging.